.changeset/calm-badgers-flash.md

```diff
@@ -0,0 +1,24 @@
+---
+"generated-test-app": patch
+"rainbowkit-next-app": patch
+"with-next-siwe-iron-session": patch
+"with-next-rainbow-button": patch
+"with-next-siwe-next-auth": patch
+"with-next-custom-button": patch
+"with-next-wallet-button": patch
+"with-next-app-i18n": patch
+"with-next-mint-nft": patch
+"with-next-app": patch
+"with-next": patch
+"example": patch
+"site": patch
+---
+
+Updated the following packages:
+
+- `next` to `^14.1.0`
+- `eslint-config-next` to `^14.1.0`
+- `next-intl` to `^3.9.1`
+
+
+For a follow up update we've updated the `with-next-app-i18n` to use the latest internalization configuration.
```

Comment:
@KosmosKey Will need to create another changeset for `create-rainbowkit` to use the new template too